frontend/packages/dev-console/src/components/topology/components/nodes/BaseNode.scss

```diff
@@ -36,7 +36,17 @@
 .odc-m-drag-active {
   .odc-base-node__contents {
     opacity: 0.3;
-    &.is-dragging, &.is-droppable {
+    &.is-dragging,
+    &.is-droppable {
+      opacity: 1;
```

Comment:
why is opacity: 1 used instead of a removeClass/style as opacity defaults to 1.

Reply:
Opacity of a base node is set to `0.3` while the user is dragging in the canvas, opacity will be set to `1` for the nodes which are eligible to drop.